.github/workflows/linters.yml

```diff
@@ -1,56 +1,21 @@
-name: Linters
-
-on: pull_request
-
-env:
-  FORCE_COLOR: 1
-
-jobs:
-  lighthouse:
-    name: Lighthouse
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
```

Comment:
Your linters are still not working. After pushing to github, github should automatically check your code and if passing give you a green marker. 
<img width="873" alt="Screenshot 2022-10-29 at 16 41 42" src="https://user-images.githubusercontent.com/48546992/198834847-6c650373-9b73-4ab1-9886-a555d6cbdecc.png">

Currently no checks done for your project. 
<img width="948" alt="Screenshot 2022-10-29 at 16 42 50" src="https://user-images.githubusercontent.com/48546992/198834937-085dbb15-cc6f-4d22-8cd3-cc9a7c998c4b.png">
 I suggest you remove the .github.workflows, .stylelintrc.json and the .hintrc and add them again.
Copy the files again from [here](https://github.com/microverseinc/linters-config/tree/master/html-css) to your project. Avoid doing it manually since this may be what is causing you problems,